lib/foreman/cast.rb

```diff
@@ -0,0 +1,22 @@
+module Foreman
+  module Cast
+    def self.to_bool(value)
+      case value
+
+      when String
+        return true if value =~ (/^(true|t|yes|y|1)$/i)
+        return false if value.blank? || value =~ (/^(false|f|no|n|0)$/i)
```

Comment:
Rails checks for on/off too:

https://github.com/rails/rails/blob/3-2-stable/activerecord/lib/active_record/connection_adapters/column.rb#L8-L9